platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/spi/WritableQueueStateBase.java

```diff
@@ -84,9 +84,7 @@ public final void commit() {
     }
 
     /**
-     * {@inheritDoc}
-     *
-     * <p>Clears the set of peeked values and added values.
+     * Clears the set of peeked values and added values.
```

Comment:
This does not appear to be necessary, and it changes the intended documentation.

Reply:
The method does not override another method, hence `{@inheritDoc}` is not allowed.